web/src/components/vis/WilcoxonPlotTile.vue

```diff
@@ -3,6 +3,7 @@ import WilcoxonPlot from '@/components/vis/WilcoxonPlot.vue';
 import VisTileLarge from '@/components/vis/VisTileLarge.vue';
 import ToolbarOption from '../ToolbarOption.vue';
 import plotData from './mixins/plotData';
+import { SET_DATASET_SELECTED_COLUMNS } from '../../store/actions.type';
```

Comment:
Perhaps change this variable name to `SET_ANALYSIS_SELECTED_COLUMNS`.  I got this confused with `SET_SELECTION` for the cleanup table.